service/matching/matcher_data.go

```diff
@@ -243,15 +245,17 @@ type matcherData struct {
 	pollers pollerPQ
 	tasks   taskPQ
 
+	logger     log.Logger
```

Comment:
should go right under `config` (always constant dependencies first, then mutable stuff)

Reply:
👍 I didn't know about that best practice.

Reply:
I'm not sure it's that well-established, I try to follow it in new code and sometimes refactor old. it makes the scope of locks and things more explicit (constants above a lock, things protected by it below, etc.)